### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by flands**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,53 +44,58 @@ $ java -javaagent:./splunk-otel-javaagent.jar -Dotel.resource.attributes=service
...
-manipulation and configures an OpenTelemetry-compatible tracer to capture
-and export trace spans. The agent also registers an OpenTelemetry `getTracer`
-so you can support existing custom instrumentation or add custom
-instrumentation to your application later.
+### Supported Java Versions
 
-To see the Java Agent in action with sample applications, see our
-[examples](https://github.com/signalfx/tracing-examples/tree/master/opentelemetry-tracing/opentelemetry-java-tracing).
+The agent works with Java runtimes version 8 and higher and supports all
+JVM-based languages (for example, Clojure, Groovy, Kotlin, Scala). Supported
```
Comment (jkwatson) [inline README.md:76]: Out of curiousity, has anyone tried using otel with Clojure? I'd love to see an example of that in action.

File: README.md [hunk 2]
```diff
@@ -117,46 +118,41 @@ A simple wrapper for the Jaeger exporter of [opentelemetry-java](https://github.
...
+the
+[Resource](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/resource/sdk.md))
+contexts.
 
 Documentation on how to inject trace context into logs is available
 [here](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/master/docs/logger-mdc-instrumentation.md).
 
-To log resource context, Splunk distribution exposes resource attributes as system properties prefixed with `otel.resource.`
-which can be used in logger configuration.
+To log resource context, Splunk distribution exposes resource attributes as
```
Comment (jkwatson) [inline README.md:142]:
```suggestion
To log resource context, the Splunk distribution exposes resource attributes as
```

File: README.md [hunk 3]
```diff
@@ -117,46 +118,41 @@ A simple wrapper for the Jaeger exporter of [opentelemetry-java](https://github.
...
 ## Correlating traces with logs
 
-To correlate traces with logs it is possible to add both trace (`traceId` and `spanId`) and 
-resource (e.g., `service.name` and `environment` attributes of the 
-[Resource](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/resource/sdk.md)) contexts.
+To correlate traces with logs it is possible to add both trace (`traceId` and
+`spanId`) and resource (e.g., `service.name` and `environment` attributes of
+the
+[Resource](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/resource/sdk.md))
+contexts.
```
Comment (jkwatson) [inline README.md:137]: this sentence is pretty mangled. I'm not sure what it's trying to communicate.

File: README.md [hunk 4]
```diff
@@ -117,46 +118,41 @@ A simple wrapper for the Jaeger exporter of [opentelemetry-java](https://github.
 | ------------------------------------ | ----------------------------------   | -------------- | -------------------------------------------------------- |
 | splunk.context.server-timing.enabled | SPLUNK_CONTEXT_SERVER_TIMING_ENABLED | false          | Enables adding `Server-Timing` header to HTTP responses. |
 
-### More options
-For more options see [here](https://github.com/open-telemetry/opentelemetry-java-instrumentation#configuration-parameters-subject-to-change)
-
-## Span tags the Splunk distribution automatically creates
```
Comment (iNikem) [inline README.md:123]: I vaguely remember somebody wanted to see all the attributes that our distribution adds on top of the standard Otel...

**Review by iNikem — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (mateuszrzeszutek) [reply README.md:123]: Maybe we can move them to a separate doc and reference it from the main README?

**Review by mateuszrzeszutek — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (flands) [inline README.md:137]:
```suggestion
To correlate traces with logs it is possible to add the following metadata from traces to logs:

 - Trace: `traceId` and `spanId`
 - Resource: `service.name` and `environment`
```

**Pull request merged**